### ISSUE TITLE
Correcting the Use Case commands' Time format and other options

### ISSUE THREAD

**Issue by thanh-lam** [IBM/CAST#327]
**Describe the bug**
The CAST online doc. Section for Python Guide has code examples that prints out command help messages with the -h option. The Time Format for those options like --starttime, --endtime, or -T is shown as YYYY-MM-DD HH:MM:SS.

That format is not accepted by the scripts.

Other command options like --hostnames for most scripts and --keywords for findJobKeys.py require better explanation. 

**To Reproduce**
1. Go to '/opt/ibm/csm/bigdata/python/'
2. export CAST_ELASTIC=<host>:9200
3. Run each script with the -h option
4. Check the output displayed
5. Also check the --hostnames or -H option from the -h output
6. For the findJobKeys.py, check the -k or --keywords option

**Expected behavior**
The Time Format requires a "T" in place of the blank between YYYY-MM-DD and HH:MM:SS. For example: 2018-10-11T13:30:00. These scripts' -h option need to be corrected: findJobsRunning.py, findJobsInRange.py, and findUserJobs.py.

For the --hostnames, add further information about the hosts where the CSM transaction logs are coming from. These hosts are not part of the allocation or job data.

For the --keywords, clarify that they will be searched in the data of the hosts' syslogs, not the transaction or allocation data.

**Screenshots**

**Environment (please complete the following information):**
 - Machine bd01 or bd02 (IST)
 - Version CSM 1.3.0

**Additional context**

**Issue Source:**
These were found during IST of the Use Cases when following the CAST online doc.

**Comment by mew2057** [comment]
```-bash-4.2$ ./findJobsRunning.py -T "2018-08-14 14:59:21.743591"
Search found 2 jobs running at '1534273161', displaying 2 jobs:

Allocation ID | Prim. Job ID | Second. Job ID | Begin Time                 | End Time
          309 |      4242442 | 0              | 2018-08-14 14:59:21.743591 | None
          310 |       900012 | 0              | 2018-08-14 15:36:31.671928 | None
-bash-4.2$```

Is the timestamp being specified with quotes surrounding the timestamp?

**Comment by thanh-lam** [comment]
The option flags --starttime, --endtime, -T for the scripts takes the timestamp without quote. For example:

<[root@bd01 ~]# findJobsInRange.py --starttime 2018-10-11T17:21 --endtime 2018-10-11T20:21
# Search found 14 jobs running, displaying 14 jobs:

User Name  | Allocation ID | Prim. Job ID | Second. Job ID | Begin Time                 | End Time
...
uno        |          1427 |         3548 | 0              | 2018-10-11 17:20:28.378999 | 2018-10-11 17:21:22.013402
uno        |          1428 |         3567 | 0              | 2018-10-11 17:21:57.97047  | None
uno        |          1429 |         3570 | 0              | 2018-10-11 17:23:39.687381 | 2018-10-12 04:41:16.770526
protocol   |          1430 |         3574 | 0              | 2018-10-11 17:30:15.027203 | 2018-10-11 22:21:37.822143>

**Comment by mew2057** [comment]
I have a fix in Pull Request #329

**Comment by thanh-lam** [comment]
Verified that use case python scripts have -h or --help for time values with correct format. For example:
```
# findJobsInRange.py -h
usage: findJobsInRange.py [-h] [-t hostname:port]
                          [--starttime YYYY-MM-DDTHH:MM:SS]
                          [--endtime YYYY-MM-DDTHH:MM:SS]
                          [-H [host [host ...]]] [-s size]
...
``` 
CSM BDS level: 1.3.0 1401.
